tests/unit_tests/test_wave_03.py

```diff
@@ -130,8 +130,4 @@ def test_swap_items_from_their_empty_returns_false():
     nobodys_item = Item()
 
     result = fatimah.swap_items(jolie, item_b, nobodys_item)
-
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
+    assert result == False
```

Comment:
This is a great start! Are there any other asserts you could check to flesh this test out just a bit?